FF1Lib/FF1Rom.cs

```diff
@@ -1183,7 +1202,20 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 
 			if (flags.TournamentSafe || preferences.CropScreen) ActivateCropScreen();
 
-			WriteSeedAndFlags(seed.ToHex(), Flags.EncodeFlagsText(flags));
+			String seedHex = seed.ToHex();
+			String encodedFlagText = Flags.EncodeFlagsText(flags);
+			WriteSeedAndFlags(seedHex, encodedFlagText);
+
+			Blob digestJson = Encoding.ASCII.GetBytes(JsonConvert.SerializeObject(new RandomizerDigest
+			{
+				seed = seedHex, 
+				encodedFlagString = encodedFlagText, 
+				commitSha = FFRVersion.Sha, 
+				inputRomSha1 = inputRomHash
+			}));
+			PutInBank(0x1B, 0xA000, digestJson);
+			//It's a little wont, but I couldn't get the blob size. So I made it a string and got the length of that
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.storage.blob.blobproperties.length?view=azure-dotnet-legacy

Reply:
Also isn't the hex a hex string? Wouldn't that be nearly twice as long?

Reply:
Pardon me it's not the .NET blob, it's the ROMUtilities one that Entroper provided.
https://github.com/Entroper/RomUtilities/blob/master/RomUtilities/Blob.cs

It still has the property!

Reply:
When I tried this before I was getting compilers, but that was also around the same time that autocomplete wasn't working. So may have just been that the environment wasn't quite set up properly I'll try that again tonight.